src/ui/index.js

```diff
@@ -10,15 +10,17 @@ import {i18n} from '../lib/native/I18n'
 Vue.mixin(i18nPlugin)
 Vue.mixin(capacitor)
 
-i18n.setLocales(navigator.languages)
-i18n.load()
-
-const app = (global['Floccus'] = new Vue({
-  el: '#app',
-  store,
-  router,
-  vuetify,
-  render: (h) => h(App),
-}))
+const app = new Promise(() => {
+  i18n.setLocales(navigator.languages)
+  await i18n.load()
+  
+  global['Floccus'] = new Vue({
+    el: '#app',
+    router,
+    store,
+    vuetify,
+    render: (h) => h(App),
+  })
+})
 
 export default app
```

Comment:
This is a bit awkward, as app is just a promise now. I guess we can remove it

Reply:
I don't have time to fully understand all the code of floccus, so try to keep the original code.
But if some code is unnecessary, it's good to remove it, we can make it cleaner.